ui/jest.config.js

```diff
@@ -0,0 +1,8 @@
+module.exports = {
+  preset: 'ts-jest',
+  testEnvironment: 'node',
+  testPathIgnorePatterns: [
+    "<rootDir>/build/",
+    "<rootDir>/node_modules/"
+  ]
+};
```

Comment:
Is it worth setting a glob for test files as `/**/*.test.*`? Or maybe that's default...I can't remember.